### DISCUSSION TITLE
MobileRaker not working, uninstall fails. K1Max fm1.3.3.29 v6.2.0

### DISCUSSION THREAD

**Discussion by riktanius** [Guilouz/Creality-Helper-Script-Wiki#598] [Issues]
I am trying to verify MobileRaker and it wasnt working. I tried to install but it says it is already installed. I tried to uninstall and I get this:

![image](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/57732530/9a0d828d-14fa-42d6-8591-4c629a1176b0)

Not sure where to go from here other than a full reset.

**Comment by Guilouz** [comment]
In SSH run this:

sudo rm -rf /usr/data/mobileraker_companion
sudo rm -rf /usr/data/mobileraker-env

And retry install.

**Comment by skbtwiz** [comment]
I just ran into the same issue so I decided to to navigate to `/usr/data/` using `cd` to check out what is going on. There I found that `mobileraker_env` directory is missing. My first guess would be that the install scripts terminate abruptly before the python environment for the companion is installed, without giving a clear indication to the user that something went wrong (just showed me a terminal prompt instead).

I then went looking in the `mobileraker_companion` directory (also in `\usr\data`) and found the install and uninstall shell scripts that you can run using the `sh` command to first cleanly uninstall, and then reinstall the Mobileraker companion.

Everything is working as intended on my end here now.
